samples/unit-tests/axis/label-ellipsis/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test("Ellipsis should be reset after zoom", function (assert) {
+QUnit.test("Ellipsis should be reset after zoom (#4678)", function (assert) {
     var chart = $('#container').highcharts({
         chart: {
```

Comment:
This test (#5034) fails because the chart container's width is set to 500px in `demo.html`, but that is not used with the karma template. Fix it by setting the `chart.width` setting explicitly in `demo.js`.